tests/integrationtest/t/expression/cast.test

```diff
@@ -1,13 +0,0 @@
-# TestNegFloatConvertToUnsigned
```

Comment:
ok so why are these 2 tests removed

Reply:
These `*.test` and `*.result` files in the `integrationtest` are not used in 7.1.